content/legal/privacy.md

```diff
@@ -1,13 +1,9 @@
 ---
-title: "Privacy"
-date: 2018-06-07T15:26:26-07:00
+title: "Privacy Policy"
+url: "/privacy"
```

Comment:
Why is `privacy` staying in the root, but all the other ones are now going to be outputted under `/legal`? (We should check with Eric if any of these need to remain in the root for some reason. If we do end up moving them to be outputted under `/legal`, I'll have need to add redirects from the old location in the root to the new location under `/legal`.

Reply:
Oh, that was unintentional. I wanted to create some navigation between them, but it seems better to leave them where they are, yes. I'll see if i can make that work.

Reply:
All right, fixed up. Thanks for the catch!